### ISSUE TITLE
Tweak JSON help message

### ISSUE THREAD

**Issue by vpetersson** [Screenly/cli#13]
This is currently how the help for JSON is displayed:

```bash
$ screenly --help
[...]
  -j, --json <JSON>  Enables json output
[...]
```

This is incorrect. `<JSON>` implies that there is some sort of input here, while in reality it just formats the output to JSON.

The output should read:

```
  -j, --json   Enables JSON output
```

**Comment by vpetersson** [comment]
While we are at it, let's also be consistent with periods at the end of sentences.

**Comment by sergey-borovkov** [comment]
Will be fixed when #15 is merged.
